multichain-testing/package.json

```diff
@@ -100,7 +102,8 @@
     "@agoric/zoe": "portal:../packages/zoe",
     "@agoric/zone": "portal:../packages/zone",
     "@cosmjs/proto-signing": "0.33.1",
-    "@cosmjs/stargate": "0.33.1"
+    "@cosmjs/stargate": "0.33.1",
+    "@agoric/pola-io": "portal:../packages/pola-io"
```

Comment:
I don't see `@agoric/pola-io` used anywhere in this PR. help me understand this change?